vdirsyncer/storage/ldap.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+import ldap3
+import vobject
+
+from .base import Item, Storage
+from .. import exceptions
+
+ldap_logger = logging.getLogger(__name__)
+
+
+class LDAPStorage(Storage):
+    '''
+    :param uri: LDAP URI
```

Comment:
URI is the right term for this since it can be an URL, a simple ip or a
hostname.

-- 
Paul Fariello

PGP: 0x672CDD2031AAF49B